libs/rays/src/PathTracer.cpp

```diff
@@ -76,8 +75,8 @@ static float3 randomPerp(float3 n) {
 namespace filament {
 namespace rays {
 
-PathTracer::Builder& PathTracer::Builder::renderTarget(image::LinearImage target) {
-    mConfig.renderTarget = target;
+PathTracer::Builder& PathTracer::Builder::outputPlane(OutputPlane target, LinearImage image) {
```

Comment:
Might be good to note which `OutputPlane`s are required and which are optional? Looks like `AMBIENT_OCCLUSION` is the only required one for now.

Reply:
Done